index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="/index.css">
+</head>
+<body>
+    <div class="wrap">
+        <header>
+            <div class="container">
+                <div class="icon"></div>
+                <a href="../login/login.html" class="login"> 로그인</a>
```

Comment:
❗️ 수정요청
공백은 css 상 의도하지 않은 결과를 만들 수 있습니다~ 불필요한 공백은 제거해주세요~

```suggestion
                <a href="../login/login.html" class="login">로그인</a>
```